main.nf

```diff
@@ -117,13 +121,12 @@ workflow {
 		.concat(custom_header_infos.map{file -> tuple(file.name.take(file.name.lastIndexOf('-custom_headers.hdf5')), file)})
 		.groupTuple()
 
-	
 	combined_metrics = combine_metric_hdf5(hdf5s_per_run, main_outdir)
 
 	// Visualize the results (and move them to the results folder)
 	visualize_results(combined_metrics)
 
-	output_processing_success(raw_files, hdf5s_per_run.toList().transpose().first().flatten())
+	output_processing_success(raw_files.concat(input_mzml_files), hdf5s_per_run.toList().transpose().first().flatten())
```

Comment:
In the `output_processing_success` I would suggest to rename `raw_files` to `spectra_files` to avoid missunderstandings. But thats it.

Reply:
Changed it to `spectra_files`